index.html

```diff
@@ -1,5 +1,12 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title>Let's get arrested</title>
 <script>
   while (1) {
       alert("!");
   }
 </script>
+</head>
```

Comment:
@ykore52 you could at least put the `<script>` on a new `<body>(...)</body>`

Reply:
According to [webarchive](http://web.archive.org/web/20190304154409/n41050z.web.fc2.com/burakura.html) of the website which has been posted by the school kid, The `<script>` was inside the `<head>`. Because of it, the modal was opened before the page was loaded. I think we should follow the original (to be arrested😅).

Reply:
Ok for me.